code/modules/food&drinks/kitchen machinery/gibber.dm

```diff
@@ -183,11 +184,15 @@
 		else
 			typeofmeat = /obj/item/weapon/reagent_containers/food/snacks/meat/slab/human
 			typeofskin = /obj/item/stack/sheet/animalhide/human
-	else
-		if(iscarbon(occupant))
-			var/mob/living/carbon/C = occupant
-			typeofmeat = C.type_of_meat
-			gibtype = C.gib_type
+	else if(iscarbon(occupant))
+		var/mob/living/carbon/C = occupant
+		typeofmeat = C.type_of_meat
+		gibtype = C.gib_type
+		if(ismonkey(C))
```

Comment:
Only humans have species datums. The rest of carbons don't even have a typeofskin var.